app-editors/levee/files/levee-4.0-tgetent-ncurses-gentoo.patch

```diff
@@ -0,0 +1,19 @@
+On Gentoo the tinfo library is built out of the libncurses source tree,
```

Comment:
Technically, you can't rely on this, and we should use pkg-config for it:
`pkg-config --libs ncurses`
or rather something like:
`${PKG_CONFIG:-pkg-config} --libs ncurses`